app/lib/actions/bulk_action.rb

```diff
@@ -9,7 +9,8 @@ class BulkAction < Actions::ActionWithSubPlans
     #   Arguments that all the targets share
     def plan(action_class, targets, *args, concurrency_limit: nil, **kwargs)
       check_targets!(targets)
-      limit_concurrency_level!(concurrency_limit) if concurrency_limit
+      extracted_concurrency_limit = extract_concurrency_limit(args, concurrency_limit)
+      limit_concurrency_level!(extracted_concurrency_limit) if extracted_concurrency_limit
```

Comment:
I think the concurrency_limit keyword is not used anywhere yet, so we might be able to change the interface if that would help

Reply:
Now I'm confused. It must have worked somehow otherwise the test that is now failing on 2.7 would never pass have passed, no?